src/option_parser.c

```diff
@@ -203,16 +203,17 @@ int ini_get_bool(const char *section, const char *key, int def)
 
 char *clean_value(const char *value)
 {
+        size_t len = strlen(value);
         char *s;
 
-        if (value[0] == '"')
-                s = g_strdup(value + 1);
+        if (value[len-1] == '"')
```

Comment:
Rather than checking this just add `-1` to the len directly, we're not using it anywhere else anyway.

Reply:
Sorry, but I don't get what you want to tell me here.